### ISSUE TITLE
Icon Request: HD icon

### ISSUE THREAD

**Issue by bernaferrari** [feathericons/feather#557]
I thought it would be nice to have a HD icon, like x-square and many others.
I did really on Illustrator something like this:
![image](https://user-images.githubusercontent.com/351125/54100202-17840d00-439c-11e9-82c8-9c5ef122a069.png) (ignore the black `D`, I don't know how to make it intersect)

Inspired by this:
https://material.io/tools/icons/?search=hd&icon=hd&style=outline

* Icon name: HD (High Definition)
* Use case: Video player
* Screenshots of similar icons: 

I think if you can get the margins correctly, it can become a really nice icon and not hard to make.

**Comment by locness3** [comment]
#171 @bernaferrari

**Comment by bernaferrari** [comment]
I'm not a designer, I barely know how to move things on illustrator. I attached everything so someone with more knowledge can make it aligned to the guidelines.

**Comment by qwerbycom** [comment]
That's a good suggestion for a font icon.

**Comment by bernaferrari** [comment]
Pixel perfect: 

![image](https://user-images.githubusercontent.com/351125/54710172-d58a6200-4b25-11e9-9fbf-d888da3350f7.png)

Do this pass the 'no fill' rule? There is no (decent) way of making the HD with strokes. I got the HD from Material Design icons and modified the D with an outside 2px corner to match the specs. If you are satisfied, I can submit it as PR.
[hd-icon.zip](https://github.com/feathericons/feather/files/2989629/hd-icon.zip)

**Comment by johnletey** [comment]
@locness3 @colebemis @bernaferrari How does [this](https://www.figma.com/file/D8WTwnQfcEvapaepx3QQed/hd) look? I followed #171

**Comment by locness3** [comment]
> @locness3 @colebemis @bernaferrari How does [this](https://www.figma.com/file/D8WTwnQfcEvapaepx3QQed/hd) look? I followed #171

It looks great, although idk if using corner radius is allowed. What do you think @colebemis ?

**Comment by johnletey** [comment]
@locness3 I personally think without the corner radius it looks like H0 and not HD ... but what is your opinion @colebemis?

**Comment by bernaferrari** [comment]
I'm not sure what's wrong with the one I sent a few months ago.

**Comment by johnletey** [comment]
@bernaferrari It didn't follow #171 ... the team is really strict about new icons following these rules!

**Comment by bernaferrari** [comment]
You could have mentioned this a few months ago.. I tried to follow everything, it is 2px everywhere.

**Comment by johnletey** [comment]
@bernaferrari Yes ... it appears you did follow the 2px rule ... but there is also the rule of rounded edges (among others) ... do go and read #171 ... and ask me any questions if anything doesn't make sense ... it personally took me a while to understand what they meant!

**Comment by ahtohbi4** [comment]
I suppose you try to get something like that:

<img width="498" alt="Снимок экрана 2019-07-01 в 14 44 05" src="https://user-images.githubusercontent.com/3686790/60434024-d9428000-9c0e-11e9-81f0-85afe2d02807.png">

<img width="541" alt="Снимок экрана 2019-07-01 в 14 43 46" src="https://user-images.githubusercontent.com/3686790/60434035-de9fca80-9c0e-11e9-9478-28dfd429ce1d.png">

```svg
<svg
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <path d="M14 9H16C17.1046 9 18 9.89543 18 11V13C18 14.1046 17.1046 15 16 15H14V9Z" />
  <path d="M6 9L6 15" />
  <path d="M10 12H6" />
  <path d="M10 9L10 15" />
  <rect x="2" y="2" width="20" height="20" rx="2" />
</svg>
```

**Comment by johnletey** [comment]
@ahtohbi4 How is this different from the one that I made?

**Comment by ahtohbi4** [comment]
@johnletey from [this one](https://github.com/feathericons/feather/issues/557#issuecomment-474976138)?

**Comment by locness3** [comment]
@ahtohbi4 No, [this](https://github.com/feathericons/feather/issues/557#issuecomment-506928445) one

**Comment by johnletey** [comment]
Thanks @locness3!

**Comment by bernaferrari** [comment]
I really liked that @ahtohbi4. Your H is less spaced.

**Comment by johnletey** [comment]
Got it @bernaferrari!

**Comment by ahtohbi4** [comment]
> @ahtohbi4 How is this different from the one that I made?

<img width="738" alt="Снимок экрана 2019-07-01 в 22 32 38" src="https://user-images.githubusercontent.com/3686790/60461789-41658600-9c50-11e9-8dbb-2a17b1265e85.png">

**Comment by ahtohbi4** [comment]
@bernaferrari @johnletey if you wish, I could create a PR with this version?

**Comment by johnletey** [comment]
@ahtohbi4 Go for it! You guys didn't like my version (and I personally like yours better 😄)!
